dev/bots/test/analyze_sample_code_test.dart

```diff
@@ -7,35 +7,42 @@ import 'dart:io';
 import 'common.dart';
 
 void main() {
-  test('analyze_sample_code', () {
+  // These tests don't run on Windows because the sample analyzer doesn't
+  // support Windows as a platform, since it is only run on Linux in the
+  // continuous integration tests.
+  if (Platform.isWindows) {
```

Comment:
Why not use the regular `skip: Platform.isWindows` on the individual tests or a test group?

Reply:
Just because this was more global, I could attach the comment about why to one location. I could have used skip on each one just as well.